### ISSUE TITLE
Give shapes x and y properties on copy/paste

### ISSUE THREAD

**Issue by madduccino** [stevekrouse/WoofJS#568]
It'd be good to have x and y properties when copying and pasting shapes so that beginner students can more intuitively adjust location.

**Comment by stevekrouse** [comment]
Good idea. This will be a quick change [here](https://github.com/stevekrouse/WoofJS/blob/master/docs/index.html). The main question is: should every default object in the Woof docs come with an `x` and `y` parameter? There are many others, including most images, that do not.

**Comment by madduccino** [comment]
Yes!

**Comment by truthgoddess** [comment]
I agree. Seems helpful. We can emphasize adding properties to objects in other ways in the curriculum.
